### PR TITLE
feat: moved markdown text to separate file, which is loaded when the …

### LINKED ISSUE #17: Rework standard md file
The standard markdown file that is displayed at the start should explain the use of the website (be a user guide).
At the same time, the markdown should be moved from the html file to a seperate markdown file

### PR DESCRIPTION
**Pull request by Szlemens00**
…website is loaded

### CODE REVIEW

File: site/index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 var js = import("../pkg/md_collab.js");
 
+var md = document.getElementById("md_input");
+
+window.addEventListener("load", read_md_file("index.md"));
```
Comment (NelDav) [inline site/index.js]: Please use camelcase for function names otherwise the js linter will fail

File: site/index.js [hunk 2]
```diff
@@ -37,3 +39,21 @@ function checkForTab(e) {
     md.selectionEnd = start + 1;
   }
 }
+
+function read_md_file(file)
+{
+    var rawFile = new XMLHttpRequest();
```
Comment (NelDav) [inline site/index.js]: In the functions above variables are declared with let. I would suggest to use  let in situations where it doesn't matter if you use var or let.

**Review by NelDav — CHANGES_REQUESTED** [review]

File: site/index.js (hunk 1 above)
Comment (Szlemens00) [reply site/index.js]: ok good point

File: site/index.js (hunk 2 above)
Comment (Szlemens00) [reply site/index.js]: okay good point

**Review by NelDav — APPROVED** [review]

**Pull request merged**